lib/screens/home/pages/requests/bloc/requests_page_bloc.dart

```diff
@@ -45,7 +45,8 @@ class RequestsPageBloc extends Bloc<RequestsPageEvent, RequestsPageState> {
       try {
         List<Relation> relations = await relationRepository.getAllRelationsAndRequests();
         yield RequestsPageSuccess(relations);
-      } on Failure catch (_) {
+      } on Failure catch (failure) {
+        Logger.root.severe(failure.message);
```

Comment:
`Logger.root.severe(failure.message);` -> `Logger.root.severe("RequestsPageBloc: Failure catched: failure.message`);`

Do this in similar places too^

We need to find a better way to log stuff but for now, let's just do logging like I've shown above

Reply:
done